tests/fixtures/backups_response.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": 1,
+    "database": "profserv2",
+    "type": "user",
+    "started_at": "something",
+    "completed_at": "something",
+    "flags": "",
+    "environment": "",
+    "_links": ""
+  }
+]
```

Comment:
Do you actually need a new line at EOF?

Reply:
AFAIK, best practice is to always end files with a new line. This is [explicitly true for Drupal](https://www.drupal.org/docs/develop/standards/coding-standards#indenting), not sure about other ecosystems.